README-Vagrant.md

```diff
@@ -0,0 +1,52 @@
+# Using Vagrant for Local Development
+
+If you don't want to install Jekyll, you can use Vagrant to set up a clean environment with all of the project's necessary dependencies.
+
+## Prerequisites
+- Vagrant: [download directly](https://www.vagrantup.com/downloads.html), `brew cask install vagrant`, or `sudo apt-get install vagrant`
+- VirtualBox: [download directly](https://www.virtualbox.org/wiki/Downloads), `brew cask install virtualbox`, or `sudo apt-get install virtualbox`
+
+## macOS Setup
```

Comment:
If we are to have this, it should be under a heading such as `OS Specific Instructions`, and we should have Linux instructions as well.

Also, maybe @ndintenfass can comment here, but if they want to use Brew to install these tools, they likely already know how to install/already have brew installed.

If they are the type of user to need instructions on installing it (working with xcode, etc) then 1) I think we should link to their instructions and 2) They probably don't want/need brew and should just install these tools from the websites, like most users who don't already use Brew would.